diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -303,10 +306,23 @@ fun ASTNode?.isAccessibleOutside(): Boolean =
             true
         }
 
+fun ASTNode.hasSuppress(warningName: String): Boolean {
+    return parent({ node ->
+        node.findChildByType(MODIFIER_LIST)
+                ?.findAllNodesWithSpecificType(ANNOTATION_ENTRY)
```

Comment:
```suggestion
                ?.findAllNodesWithSpecificType(ANNOTATION_ENTRY)
                ?.map { it.psi as KtAnnotationEntry }
```
to avoid a lot of casts later